### ISSUE TITLE
multiple budgets per site

### ISSUE THREAD

**Issue by babybunny** [babybunny/rebuildingtogethercaptain#370]
For example, currently a site can be funded by multiple funding pools (CDBG, private, roof fund), 

There should be multiple funding pools in the system, and each site can have a budget from each. 

Ideally have three fields on Site Info, (roof budget, CDBG budget, private budget) and Jo can update budgets separately. 

Jo also would like an internal notes field for keeping track of the reasons for the budget changes.

**Comment by babybunny** [comment]
Looking for some screenshots and/or drawings for where these additional numbers should appear. 

Sounds like there is an excel that is currently used for Jo's purpose. Can you screenshot it so I can see the columns and example values?

**Comment by jophillipsrtp** [comment]
Let me try and explain my main problem with the Budget. On MY end, there is no break down of where the Budget is coming from (Funding sources). I would like the Edit site info page to show ME 4 different Funding Source options (CDBG, Sponsorship, Roof, Other) where I CAN fill in different amounts for the different Funding sources. This does not need to be connected to invoices or anything else. Nothing else needs to be changed in regards to what the captains will see or do- nothing should be different for them, and when they submit any receipt it should automatically take from the "Captains Budget". Is it possible to make all 4 different funding sources automatically add together to create a "Total site Budget" (if not, no problem, I can add and put in the correct amount) which would be different than the "Captain's Budget" (This Captain Budget would be what the captains see and what all expense automatically (on the captain's end) subtract from (They never should need to pick what budget its coming from). (Helpful to understand- anything a captain orders would come from only 2 budgets- the CDBG Budget and sponsorship Budget (these are the sources I will use to figure out their Captain Budget)- captains would never do anything with the "Roof" or "Other" Budget). Basically, each site would have an "Original Budget" (That I would put in once and  would never change so that I can see where we started), there would be a "Captain's Budget" (which I would put in by adding the CDBG funding and Sponsorship funding) and this is the only budget the Captain should see and all receipts they submit (just like now) would automatically come from this 1 budget, shown on their site page. Then lets say I need to add to the site's Roof budget and then also subtract from that budget by adding a roof estimate/invoice- I (only 1) would need to be able to select that specific invoice coming from the Roof Budget, but the captains should not see this nor should it affect the Captain's budget. Same goes for the "other" funding source. Sorry if it is confusing or complicated! Let me know what needs to be clarified!!

**Comment by babybunny** [comment]
In your example, is the roof invoice visible at all to the captain?

**Comment by jophillipsrtp** [comment]
Yes, It would be fine if the Captains can still see the roof invoice, similar to how they do now.

**Comment by jophillipsrtp** [comment]
I'm going to send you (email) a scanned print out of our ROOMs edit site info page to show how I would envision this. Something important to note is that there are immediate goals with how to keep track of the different budgets on ROOMs, and then there are future goals of what could be done long term to make the entire thing more useful. I noted the difference in my scan.

**Comment by babybunny** [comment]
I got it!
